src/odemis/driver/actuator.py

```diff
@@ -594,6 +594,13 @@ class ConvertStage(model.Actuator):
     the two stages, as for each SEM stage move it is able to perform the
     corresponding “compensate” move in objective lens.
     """
+    def __new__(cls, *args, **kwargs):
+        # Automatically create Convert3DStage object if the number of axes = 3
+        axes = kwargs['axes']
+        if len(axes) == 2:
+            return super(ConvertStage, cls).__new__(cls)
+        elif len(axes) == 3:
+            return super(ConvertStage, cls).__new__(Convert3DStage)
```

Comment:
We need to raise a ValueError in other cases explaining the number of axis is incorrect. Otherwise `__new__()` returns None and so `None` would be used as instance, creating a lot of confusion on what is happening.